apps/devhub/views.py

```diff
@@ -1259,6 +1260,9 @@ def version_add_file(request, addon_id, addon, version_id):
             # Passed validation: sign automatically without manual review.
             new_file.update(status=amo.STATUS_LITE)
             sign_file(new_file)
+    elif new_file_form.cleaned_data.get('beta'):
+        # Beta files always get signed with prelim cert.
+        sign_file(new_file)
```

Comment:
That's not what we want: we don't want to sign them automatically, unless they've passed automatic validation. If they don't, they should only be queue for prelim review.

We don't have a status for "beta version waiting for review", so I think we're going to need a new one. I'll post a comment in the bug.